services/docker-api-proxy/docker/boot.sh

```diff
@@ -12,4 +12,7 @@ echo "$INFO" "User :$(id "$(whoami)")"
 #
 # RUNNING application
 #
-socat TCP-LISTEN:8888,fork,reuseaddr UNIX-CONNECT:/var/run/docker.sock
+socat TCP-LISTEN:8889,fork,reuseaddr UNIX-CONNECT:/var/run/docker.sock &
```

Comment:
just to be sure: the port changed here

Reply:
Yes, this is correct, port 8889 is only used internally, 8888 is exposed